src/assets/moon.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 460 550"><defs/><path fill="#f7b000" d="M358.6 96.1c-11.9-10.4-24.9-19.6-38.9-27.2-2.7-1.5-6.2-1.1-8.5 1-2.2 2-2.6 5.3-.9 7.8 48.9 70.1 38.5 160.7-25.4 220.2-37.3 34.7-87.2 53.8-140.6 53.8-35.3.1-70.1-8.6-100.9-25.3-2.7-1.5-6.2-1.1-8.5 1-2.2 2-2.6 5.3-.9 7.8 8.9 12.7 19.2 24.3 30.9 34.6 39.5 35 91.7 54.3 146.9 54.3 57.2 0 110.7-20.5 150.5-57.7 39.3-36.6 60.5-84.8 59.8-135.9-.7-51-23.3-98.8-63.5-134.4z"/></svg>
```

Comment:
Can you replace the moon icon with following SVG? 

https://freeicons.io/ios-edge-glyph/moon-icon-29411